torch/nn/utils/parametrize.py

```diff
@@ -0,0 +1,375 @@
+from ..modules.module import Module
+from ..parameter import Parameter
+import functools
+import inspect
+
+
+class Parametrization(Module):
+    r"""A kind of Module that parametrizes a Parameter or a buffer in terms
+    of its forward function
+
+    After registering a Parametrization ``param`` on a tensor ``t`` within a module
```

Comment:
`param` sounds like a parameter, and I wouldn't call the parametrized thing just a "tensor", since it's a very special kind of a tensor (only parameter or a buffer).